_config.yml

```diff
@@ -455,6 +452,9 @@ navigation:
       - text: Time tracking and billing <br><br>
         url: https://handbook.18f.gov/tock/#time-tracking-and-billing
         internal: false
+      - text: Blogging
```

Comment:
@coreycaitlin I was also working on this file for a WIP PR and wondering what the logic is for how these child sections are ordered. Seems like they were alphabetical once upon a time, then items have been added at the end ad hoc. What say you to bringing back the alphabetical order here?

Reply:
Sure, I can do that — I just added it to the end tbh

Reply:
Done!